docs/plugins.rst

```diff
@@ -240,6 +240,35 @@ secret values into the plugin's configuration page, and hit **Save**. Users shou
 then be able to log in with their Google account when they click the log in page
 and select the option to log in with Google.
 
+Extension
+*********
+
+This plugin can also be extended to do more than just login behavior using the
+OAuth providers. For instance, if you wanted some sort of integration with a
+user's Google+ circles, you would add a custom scope that the user would have
+to authorize during the OAuth login process.
+
+.. code-block:: python
+
+    from girder.plugins.oauth.providers.google import Google
+    Google.addScopes(['https://www.googleapis.com/auth/plus.circles.read'])
+
+Then, you can hook into the event of a user logging in via OAuth. You can
+hook in either before the Girder user login has occurred, or afterward. In
+our case, we want to do it after the Girder user has been established:
```

Comment:
can you be more precise about what `established` means here, related to registration vs login?